unlock-js/scripts/compressAbi.js

```diff
@@ -76,6 +82,24 @@ export default bytecode
 `
 }
 
+// this is used to generate the bytecode for Unlock contracts that is used in deploy.js
+function formatDeployableBytecode() {
```

Comment:
Not a suggestion for our project, but if you ever find yourself doing a lot of this sort of thing you should look into `sweet.js`